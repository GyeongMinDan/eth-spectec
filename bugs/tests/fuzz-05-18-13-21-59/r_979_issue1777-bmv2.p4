 @metadata @name("standard_metadata") struct standard_metadata_t {
  }
   extern register<T> {
      register(bit<32> size);
  }
   header ethernet_t {
  }
   struct headers_t {
  }
   struct reg_data2_t {
  }
   struct metadata_t {
  }
   control ingress(inout headers_t hdr,                 inout metadata_t meta,                 inout standard_metadata_t stdmeta) {
      register<bit<8> >(16) reg1;
      apply {
     }
  }
