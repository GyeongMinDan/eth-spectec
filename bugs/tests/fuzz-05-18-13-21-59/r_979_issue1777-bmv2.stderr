likely_hits/r_979_issue1777-bmv2.p4(3): [--Wwarn=unused] warning: 'T' is unused
   extern register<T> {
                   ^
likely_hits/r_979_issue1777-bmv2.p4(14): [--Wwarn=unused] warning: Control ingress is not used; removing
   control ingress(inout headers_t hdr, inout metadata_t meta, inout standard_metadata_t stdmeta) {
           ^^^^^^^
[--Wwarn=missing] warning: Program does not contain a `main' module
